bandwidth/webrtc/utils/transfer_util.py

```diff
@@ -6,14 +6,20 @@
 @copyright Bandwidth INC
 """
 
-def generate_transfer_bxml(device_token, sip_uri='sip:sipx.webrtc.bandwidth.com:5060'):
+def generate_transfer_bxml(device_token, voice_call_id, sip_uri='sip:sipx.webrtc.bandwidth.com:5060'):
     """
     Returns BXML string with WebRTC a device token to perform a SIP transfer
+    :param device_token: The device token
+    :param voice_call_id: The voice call id, used for debugging
+    :param sip_uri: The uri of the sipx, if not specified it will take the default value
```

Comment:
Sorry to nit-pick... Customers aren't going to know what sipx is... I'd refer to it as the SIP URI to transfer the call to

Reply:
Good point, no need to apologize!